pkgs/development/python-modules/py-cpuinfo/default.nix

```diff
@@ -15,9 +17,12 @@ buildPythonPackage rec {
      sha256 = "10qfaibyb2syiwiyv74l7d97vnmlk079qirgnw3ncklqjs0s3gbi";
   };
 
+  # On Darwin sysctl is used to read CPU information.
+  propagatedBuildInputs = lib.optionals stdenv.isDarwin [ sysctl ];
```

Comment:
When propagating checkInputs shouldn't require sysctl but I tested it and the test fails. Maybe we need to hardcode the path or wrap python to set the path right?